packages/flutter_tools/test/integration.shard/timeline_test.dart

```diff
@@ -96,7 +96,7 @@ void main() {
         // The downloaded part of the timeline may contain an end event whose
         // corresponding begin event happened before the start of the timeline.
         if (stack.isNotEmpty) {
-          expect(stack.removeLast(), name);
+          expect(stack.contains, name);
```

Comment:
isn't `stack.contains` a function reference?

Reply:
or actually, even better, use the `contains()` matcher, which will show us what's in the set if it fails:

```dart
expect(stack, contains(name));
```

Reply:
I would also force this code path to hit locally so that you can confirm the matcher expectation is formed correctly